go/test/endtoend/throttler/util.go

```diff
@@ -95,6 +90,14 @@ func UpdateThrottlerTopoConfig(clusterInstance *cluster.LocalProcessCluster, ena
 	}
 }
 
+// UpdateThrottlerTopoConfig runs vtctlclient UpdateThrottlerConfig.
+// This retries the command until it succeeds or times out as the
+// SrvKeyspace record may not yet exist for a newly created
+// Keyspace that is still initializing before it becomes serving.
+func UpdateThrottlerTopoConfig(clusterInstance *cluster.LocalProcessCluster, enable bool, disable bool, threshold float64, metricsQuery string) (result string, err error) {
+	return UpdateThrottlerTopoConfigRaw(&clusterInstance.VtctldClientProcess, clusterInstance.Keyspaces[0].Name, enable, disable, threshold, metricsQuery)
```

Comment:
This is problematic when there are multiple keyspaces, no? Shouldn't we range over `clusterInstance.Keyspaces` here?